internal/testdata/common.go

```diff
@@ -38,41 +38,41 @@ const (
 )
 
 func initResourceAttributes1(dest pdata.AttributeMap) {
-	dest.InitFromMap(resourceAttributes1)
+	pdata.NewAttributeMapFromMap(resourceAttributes1).CopyTo(dest)
```

Comment:
Probably we can make the resourceAttributes1 to be AttributesMap, separate PR I am ok, but just an issue?